src/mesh/RTT_Draco_Mesh_Reader.hh

```diff
@@ -51,6 +52,11 @@ public:
   std::vector<unsigned> get_cellnodes(size_t cell) const {
     return rtt_reader->get_cells_nodes(cell);
   }
+  std::vector<unsigned> get_cellfacenodes(size_t /*cell*/,
+                                          size_t /*face*/) const {
+    Insist(false, "cell-face nodes not implemented with RTT reader.");
+    return std::vector<unsigned>();
+  }
```

Comment:
You might want to add the function decoration `[[noreturn]] ` (see `ds++/Assert.hh` for an example) to avoid any compiler confusion.

Reply:
Thank you for the recommendation - this is a useful feature.  A `[[noreturn]]` decoration has been added and the `return` line has been removed.